bodhi/server/scripts/untag_branched.py

```diff
@@ -53,15 +49,13 @@ def main(argv=sys.argv):
     log = logging.getLogger(__name__)
 
     settings = get_appsettings(config_uri)
-    engine = engine_from_config(settings, 'sqlalchemy.')
-    Session = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
-    Session.configure(bind=engine)
+    initialize_db(settings)
     db = Session()
     koji = buildsys.get_session()
     one_day = timedelta(days=1)
     now = datetime.utcnow()
 
-    with transaction.manager:
+    try:
```

Comment:
Could it be nice to use ```bodhi.server.util.TransactionalSessionMaker``` here to use the ```with``` statement? I guess it doesn't have a built-in way to do things like ```log.error()```, but maybe we could add a feature to it so you can register a failure handler that gets called when a rollback happens. That might be overkill. I'm fine with this is you prefer it, or whatever. I'm just braindumping.